jobinfo

```diff
@@ -179,6 +186,88 @@ def f_state(x, meta):
         deps = " (%s)" % deps
     return ','.join(states) + reason + deps
 
+def memory_to_bytes(memory_spec, cores_per_node):
+   # Convert memory_spec to bytes, ignoring the c or n at the end
+   memory_req = byte_size(memory_spec[:-1])
+
+   # If the memory request was per core, multiply by the average number of cores
+   # This is the best we can do given the lack of detailed information
+   if memory_spec[-1] == 'c':
+      memory_req = memory_req*cores_per_node
+   return memory_req
+
+def get_cpus_node(nodenames):
+    nodename=nodenames.split(',')[-1]
+    info = subprocess.Popen(['scontrol', 'show','node', nodename],stdout=subprocess.PIPE)
+    for line in info.stdout:
+        if 'CfgTRES' in line:
+            cpucount = re.search('cpu=(\d+)',line).group(1)
+    return cpucount
+
+def get_hints(meta):
+    if meta.end.lower() == 'unknown':
```

Comment:
This causes hints to be only printed for jobs that have finished. Is this what we want? We could change this for a check that starts printing hints if a job has been running for more than an hour.

Reply:
I think this is okay for now, we can always change this later on.